pkg/webhook/mutation/pod/v2/oneagent/config.go

```diff
@@ -0,0 +1,9 @@
+package oneagent
```

Comment:
v1 has just config.go in top level, why we use metadata for this?

Reply:
nvm, i got it. https://github.com/dynatrace/dynatrace-operator/blob/3c457aafd39e8363f24cad6da3e5f956f1055218/pkg/webhook/mutation/pod/v2/metadata/containers.go#L16